tools/convert_poses.py

```diff
@@ -0,0 +1,70 @@
+import os
+from pathlib import Path
+import json
+from pprint import pprint
+
+
+def list_video_names(videos_path, video_list_file):
+    """
+    Creates a file containing the list of all the video names for the videos in 'video_path'
+    """
+    with open(video_list_file, 'w') as f:
+        p = Path(videos_path)
+        for path in p.glob('*.avi'):
+            video_path = str(path)
+            video_name = (video_path.split('/')[-1]).split('.')[0]
+            f.write(video_name + '\n')
+
+if __name__ == "__main__":
+    # folder that contains all the video that have been treated by OpenPose
+    videos_path = '../../myprojectfolder/data/videofolder'
```

Comment:
Hi, thanks for your contribution!
Would you please change these paths to configurable arguments of this script? So other users can directly use the script without modifying the code.

The `argparse` module will be handy here. For usage you can refer to Sijie's code here

https://github.com/yysijie/st-gcn/blob/master/tools/ntu_gendata.py#L100